x-pack/plugins/observability_solution/apm/server/lib/helpers/transactions/index.ts

```diff
@@ -170,7 +170,6 @@ export function getDurationLegacyFilter(): QueryDslQueryContainer {
       must: [
         {
           bool: {
-            filter: [{ exists: { field: TRANSACTION_DURATION_HISTOGRAM } }],
```

Comment:
Don't think we need the outer `bool` and `must` now there is only the `must_not` and could do:
``` 
"bool": {
  "must_not": {
    "exists": {
      "field": TRANSACTION_DURATION_HISTOGRAM
    }
  }
}
```